lib/validation/api.js

```diff
@@ -14,9 +14,12 @@ var joi = require('joi')
 module.exports = joi.object({
     name:               joi.string()
   , request_path:       joi.string().optional()
+  , uris:               joi.array().items(joi.string().optional())
```

Comment:
If `0.9` and `0.10` don't have a compatible API, I don't think this project should try to maintain compatibility with breaking changes in the project.

Someone could Have `0.10` Kong instance running and make a valid `0.9` request. 

I'd much rather follow suite: just make the changes required to work with `0.10`, disregard `0.9`. I'll just publish a new major version.

I can set up CI to run on a tag for each Kong version. Less cruft in the code, less prone to user error and easier to deal with.

Mind just removing the old `0.9` stuff? 

Thanks!